infrastructure/deployment/deploy.sh

```diff
@@ -284,14 +312,13 @@ docker_stack_deploy() {
     fi
 
     echo "Downloading $tag"
-
-    until configured_ssh "cd /opt/opencrvs && docker pull $tag"
-    do
-      echo "Server failed to download $tag. Retrying..."
-      sleep 5
-    done
+    echo -e "$tag \n" >> docker_images.txt
```

Comment:
Is there a benefit for the extra step of first writing the `IMAGE_TAGS_TO_DOWNLOAD[@]` list to a file? Originally this loop's purpose was downloading the images, so we could keep it at that.

Reply:
This loop contains all the images which has already present in the machine